ui-tests/src/test/java/org/openmrs/reference/page/RegistrationEditSectionPage.java

```diff
@@ -23,6 +23,7 @@
     private static final By STATE_PROVINCE = By.id("stateProvince");
     private static final By COUNTRY = By.id("country");
     private static final By POSTAL_CODE = By.id("postalCode");
+    private static final String PHONE_NUMBER_VALIDATOR = "Must be a valid phone number (with +, -, numbers or parentheses))";
```

Comment:
Is this the actual validator or validation message?

Reply:
> Is this the actual validator or validation message?

What a question!

Waiting to hear @sherrif10 's response!

Reply:
> Is this the actual validator or validation message?

Its a validation message. Thanks @dkayiwa

Reply:
I thought the answer would be a followup commit. 😊

Reply:
@dkayiwa , Didnt get that clearly .

Reply:
Let me give you some time to think through my original message and see if you have to do something. 😊

Reply:
`NOTIFICATION` is confusing.

Reply:
Resolved with _PHONE_NUMBER_MESSAGE_ . How about phone_number_message.